include/astshim/FitsChan.h

```diff
@@ -485,6 +487,14 @@ class FitsChan : public Channel {
     std::string getCardName() const { return getC("CardName"); }
 
     /**
+    Get the name of all cards, in order, starting from the first card
+
+    Not "const" because the code changes the index of the current card while operating
+    (but restores the original index when done).
+    */
+    std::vector<std::string> getCardNames();
```

Comment:
Getting back to your first comment...I agree that {{getCardNames}} could be confused with {{getCardName}} though I was hoping we could live with it. Is it worth changing to {{getAllCardNames}} or do you have a better suggestion?

{{FitsChan}} makes heavy use of its index and the concept of the current card, so I think it will take more work than it's worth to make this method const.